services/pg-backup/template.env

```diff
@@ -0,0 +1,5 @@
+MACHINE_FQDN=${MACHINE_FQDN}
+POSTGRES_USER=${POSTGRES_USER}
+POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+POSTGRES_PORT=${POSTGRES_PORT}
+PREFIX_STACK_NAME=${PREFIX_STACK_NAME}
```

Comment:
you didnt run the pre-commit hooks, please do so:|
`pre-commit install && pre-commit run --all`